src/views.py

```diff
@@ -0,0 +1,68 @@
+from colors import Colors
+import os
+
+class SceneManager:
+    '''This `static` class will manage anything that
+       gets printed. Handles all renders'''
+
+    __alert__ = ''
+    __current__ = None
+
+    @classmethod
```

Comment:
I like how you broke out the views into its own class. Why did you make these class methods and why use dunder attributes?